src/components/spinner/spinner.jsx

```diff
@@ -1,29 +1,26 @@
-const range = require('lodash.range');
-const PropTypes = require('prop-types');
 const React = require('react');
+const PropTypes = require('prop-types');
+const classNames = require('classnames');
 
 require('./spinner.scss');
 
 // Adapted from http://tobiasahlin.com/spinkit/
-const Spinner = ({
-    mode
-}) => {
-    const spinnerClassName = (mode === 'smooth' ? 'spinner-smooth' : 'spinner');
-    const spinnerDivCount = (mode === 'smooth' ? 24 : 12);
-    return (
-        <div className={spinnerClassName}>
-            {range(1, spinnerDivCount + 1).map(id => (
-                <div
-                    className={`circle${id} circle`}
-                    key={`circle${id}`}
-                />
-            ))}
-        </div>
-    );
+// Available colors right now are white, blue and transparent-gray
+const Spinner = props => (
```

Comment:
I think it would be good to keep using the existing method of enumerating the used props, since it makes it clear when a prop becomes unused:
```jsx
const Spinner = ({
    className,
    color
}) => (
// ...
);
```

Reply:
Oh ok!